### DISCUSSION TITLE
Lithium + Opioids

### DISCUSSION THREAD

**Discussion by utaninja** [TripSit/drugs#154] [Questions]
This is a early start to my research on the combination Opioids + Lithium, just some links I found.

https://www.sciencedirect.com/science/article/abs/pii/0306362394903654
https://journals.sagepub.com/doi/abs/10.1177/0269881107083848?journalCode=jopa
https://pubmed.ncbi.nlm.nih.gov/1848627/
https://link.springer.com/article/10.1007/s11033-020-05740-9
https://jpet.aspetjournals.org/content/256/3/1101
https://www.longdom.org/open-access/the-effects-of-lithium-on-the-endogenous-opioid-system-a-model-for-nonsuicidal-selfinjury-behavior-100772.html
https://www.sciencedirect.com/science/article/abs/pii/030645229090012S
https://onlinelibrary.wiley.com/doi/pdf/10.1034/j.1600-0773.2002.900206.x
https://www.science.org/doi/10.1126/science.319532
https://australianprescriber.tg.org.au/articles/opioids-and-antidepressants-which-combinations-to-avoid.html
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3334287/
https://ajp.psychiatryonline.org/doi/10.1176/ajp.155.1.12

**Comment by utaninja** [comment]
Lithium + Opioids
This seems to be a complex interaction that has varying reports. Notably, although there seems to be a significant amount of research on lithium and it's interaction with opioid receptors, there does not seem to much flagging the specific drug interaction. Further insight and review would be appreciated (:

* Increase or decrease of effects?
* CNS depression concerns?

[The highs and lows of prescribing and monitoring lithium safely](https://doi.org/10.1002/psb.2034)

[Lithium: effects on subjective functioning and morphine-induced euphoria](https://doi.org/10.1126/science.319532)
Lithium is suggested to be an anti euphoric

[Influence of chronic lithium treatment upon the motivational effects of opioids: alteration in the effects of mu- but not kappa-opioid receptor ligands](https://pubmed.ncbi.nlm.nih.gov/1848627/
)
Snippets manually pulled - `The influence of chronic lithium (Li) treatment upon the secondary reinforcing effects of opioid agonists and antagonists was examined by use of an unbiased place preference conditioning procedure. Administration of the mu-agonist morphine to control rats resulted in marked preferences for the drug-associated place and a similar effect was observed in response to the psychostimulant d-amphetamine. In contrast, the selective kappa-opioid agonist U-69593 [(5 alpha,7 alpha,8 beta)-(-)-N-methyl-N-(7-1(pyrrolidinyl-1-oxaspirol(4,5) dec-8-yl benzeneacetamide)] and the opioid antagonist naloxone produced dose-related place aversions. Chronic administration of a Li-containing diet, which produced serum levels of 0.56 mmol/l, abolished the place preferences induced by morphine but not d-amphetamine. This treatment abolished the aversive effects of naloxone but did not modify those produced by U-69593. These data and those from a previous place conditioning study indicate that Li can function as an antagonist of mu-opioid receptor ligands in vivo and that this action underlies its motivational effects. Furthermore, the inability of chronic Li treatment to modify either the content or basal release of beta-endorphin in various brain regions suggests that this antagonism is mediated directly at the level of the opioid receptor and/or its transducer systems.`

[Temporal Variation in Drug Interaction Between Lithium and Morphine‐Induced Analgesia](https://doi.org/10.1080/07420520600650745)
Snippet manually pulled - `Reports about the interaction between lithium and pain perception are contradictory. Some investigators report that both the chronic and acute administration of lithium exerts analgesia in the mouse hot-plate test (Mannisto and Saarnivaara, 1972; Tulunay et al., 1971). Some other investigators report that lithium induces hyperalgesia (McNally and Westbrook, 1998; Wiertelak et al., 1994). Finally, some report no effect (Dehpour et al., 1994; Jensen, 1974). Also there are conflicting reports about the interaction between lithium and morphine-induced analgesia. Several authors suggest that lithium potentiates (Dehpour et al.,1994; Mannisto and Saarnivaara, 1972; Tulunay et al., 1971), while others report it inhibits (McNally and Westbrook, 1998) the analgesic effect of morphine. Also, it has been proposed that lithium exerts a different, biphasic effect on morphine-induced antinociception (Dehpour et al., 1994). Although the effects of lithium on pain perception are reported to be dose- and testdependent (de Gandarias et al., 2000; Dehpour et al., 1994; Raffa and Martinez, 1992), we did not observe any analgesic or algesic effect of lithium with the dose selected for our experiments.`

[Lithium attenuated the behavioral despair induced by acute neurogenic stress through blockade of opioid receptors in mice
](https://doi.org/10.1016/j.biopha.2016.08.015) Antidepressant action by lithium is suggested to be meditated by opioids

[The effect of lithium on morphine-induced analgesia in mice](https://doi.org/10.1016/0306-3623(94)90365-4)
Snippets manually pulled -
`The present results are at variance with the finding of Jensen (1974) and Staunton et al. (1982), who showed potentiation of chronic Li + administration on morphine analgesia in the hot-plate test. These authors worked with rats and the plasma and cerebral Li + levels achieved in the two studies were 1 mM and 0.7-1 raM, respectively, i.e. about twice of that achieved in the present study on mice. Therefore, the effect of Li + on analgesia can be either reduction or potentiation, depending on the species, the method of pain testing and especially dose of Li +.  
The idea that Li ÷ may modify the antinociceptive effect of opiate analgesics like morphine is supported by finding that Li + decreased the binding of opioid agonists to brain opiate receptors (Hitzemann et al., 1974; Amir and Simantov, 1981; Wajda et al., 1981; Stengaard-Pedersen and Schou, 1982, 1985; Paterson et al., 1986) and subsequently reduce the behavioural effects of injected agonists. Opiate-receptors binding is not the only system affected by Li +.`


Additional Studies:
[Lithium reverses the effect of opioids on eNOS/nitric oxide pathway in human umbilical vein endothelial cells](https://doi.org/10.1007/s11033-020-05740-9)
[Influence of chronic lithium treatment upon the motivational effects of opioids: alteration in the effects of mu- but not kappa-opioid receptor ligands.](https://jpet.aspetjournals.org/article/S0022-3565(25)23053-1/abstract)

**Comment by zelixir25** [comment]
I figured I'd check the drugs.com interaction checker on this one. It flags it as having a "moderate" risk related to serotonin syndrome. They have a reference list as well that I haven't had a chance to comb through. A lot of them seem to be related to tramadol + SSRIs, though, which doesn't seem particularly relevant here. I know there is, however, some slight risk of SSRIs + certain opioids in general so maybe that is also informative here?
https://www.drugs.com/interactions-check.php?drug_list=1477-16548,1770-2012&professional=1

**Reply by utaninja** [reply]
Saw this too. There's a general concern of certain opioids and serotonin syndrome, but the combination chart seems to lean towards the assumption it's not one of those opioids. (We discussed this when we were talking about possibly replacing opioids with oxycodone or similar)

**Comment by utaninja** [comment]
Revised write-up:
There are varying reports and studies on this interaction. Although there is a significant amount of research on lithium and its interaction with opioid receptors, there does not seem to be much flagging the specific drug interaction.

As a general note, there is a concern with certain opioids and serotonin syndrome, but the combination chart generally leans towards the assumption that it's not one of those opioids. In future development of the chart, ideally, we will turn away from using drug categories and give as many specific inter

General thoughts, do they increase or decrease each other’s effects? Are there concerns of CNS depression?
* Increase or decrease of effects?
* CNS depression concerns?

General study that has useful information - [The highs and lows of prescribing and monitoring lithium safely](https://doi.org/10.1002/psb.2034)
Study suggesting Lithium as an anti euphoric - [Lithium: effects on subjective functioning and morphine-induced euphoria](https://doi.org/10.1126/science.319532)

[Influence of chronic lithium treatment upon the motivational effects of opioids: alteration in the effects of mu- but not kappa-opioid receptor ligands](https://pubmed.ncbi.nlm.nih.gov/1848627)
Some highlights from it - The influence of chronic lithium (Li) treatment upon the secondary reinforcing effects of opioid agonists and antagonists was examined by use of an unbiased place preference conditioning procedure. Administration of the mu-agonist morphine to control rats resulted in marked preferences for the drug-associated place and a similar effect was observed in response to the psychostimulant d-amphetamine. In contrast, the selective kappa-opioid agonist U-69593 [(5 alpha,7 alpha,8 beta)-(-)-N-methyl-N-(7-1(pyrrolidinyl-1-oxaspirol(4,5) dec-8-yl benzeneacetamide)] and the opioid antagonist naloxone produced dose-related place aversions. Chronic administration of a Li-containing diet, which produced serum levels of 0.56 mmol/l, abolished the place preferences induced by morphine but not d-amphetamine. This treatment abolished the aversive effects of naloxone but did not modify those produced by U-69593. These data and those from a previous place conditioning study indicate that Li can function as an antagonist of mu-opioid receptor ligands in vivo and that this action underlies its motivational effects. Furthermore, the inability of chronic Li treatment to modify either the content or basal release of beta-endorphin in various brain regions suggests that this antagonism is mediated directly at the level of the opioid receptor and/or its transducer systems.

[Temporal Variation in Drug Interaction Between Lithium and Morphine‐Induced Analgesia](https://doi.org/10.1080/07420520600650745)
Some highlights from it - Reports about the interaction between lithium and pain perception are contradictory. Some investigators report that both the chronic and acute administration of lithium exerts analgesia in the mouse hot-plate test (Mannisto and Saarnivaara, 1972; Tulunay et al., 1971). Some other investigators report that lithium induces hyperalgesia (McNally and Westbrook, 1998; Wiertelak et al., 1994). Finally, some report no effect (Dehpour et al., 1994; Jensen, 1974). Also there are conflicting reports about the interaction between lithium and morphine-induced analgesia. Several authors suggest that lithium potentiates (Dehpour et al.,1994; Mannisto and Saarnivaara, 1972; Tulunay et al., 1971), while others report it inhibits (McNally and Westbrook, 1998) the analgesic effect of morphine. Also, it has been proposed that lithium exerts a different, biphasic effect on morphine-induced antinociception (Dehpour et al., 1994). Although the effects of lithium on pain perception are reported to be dose- and testdependent (de Gandarias et al., 2000; Dehpour et al., 1994; Raffa and Martinez, 1992), we did not observe any analgesic or algesic effect of lithium with the dose selected for our experiments.

Antidepressant action by lithium is suggested to be mediated by opioids - [Lithium attenuated the behavioral despair induced by acute neurogenic stress through blockade of opioid receptors in mice](https://doi.org/10.1016/j.biopha.2016.08.015)

[The effect of lithium on morphine-induced analgesia in mice](https://doi.org/10.1016/0306-3623(94)90365-4)
Some highlights from it - The present results are at variance with the finding of Jensen (1974) and Staunton et al. (1982), who showed potentiation of chronic Li + administration on morphine analgesia in the hot-plate test. These authors worked with rats and the plasma and cerebral Li + levels achieved in the two studies were 1 mM and 0.7-1 raM, respectively, i.e. about twice of that achieved in the present study on mice. Therefore, the effect of Li + on analgesia can be either reduction or potentiation, depending on the species, the method of pain testing and especially dose of Li +.  
The idea that Li ÷ may modify the antinociceptive effect of opiate analgesics like morphine is supported by finding that Li + decreased the binding of opioid agonists to brain opiate receptors (Hitzemann et al., 1974; Amir and Simantov, 1981; Wajda et al., 1981; Stengaard-Pedersen and Schou, 1982, 1985; Paterson et al., 1986) and subsequently reduce the behavioural effects of injected agonists. Opiate-receptors binding is not the only system affected by Li +.

[Lithium reverses the effect of opioids on eNOS/nitric oxide pathway in human umbilical vein endothelial cells](https://doi.org/10.1007/s11033-020-05740-9)

[Influence of chronic lithium treatment upon the motivational effects of opioids: alteration in the effects of mu- but not kappa-opioid receptor ligands](https://jpet.aspetjournals.org/article/S0022-3565(25)23053-1/abstract)
